RELEASING.md

```diff
@@ -0,0 +1,24 @@
+# Releasing new version
+
+## Step-by-step
+
+In order to release a new version, there are several steps that must happen:
+
+ 0. make sure all issues (in GitHub and in JIRA) targeted to the version are resolved
+ 1. create a pull request in which you change the [version string](https://github.com/modcluster/mod_proxy_cluster/blob/main/native/include/mod_proxy_cluster.h#L21) according to the format described in [another section](#version-naming-conventions)
+ 2. get review and merge the pull request
+ 3. release the version through GH release functionality with the tag corresponding to version from the pull request
```

Comment:
The "GH release functionality" doesn't tag. Just creates a page for the tag.

We are missing the `git tag` + push step.

Reply:
You can create a tag directly through it.

Reply:
See here https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository?tool=webui (it's not the most intuitive, but it works).

Reply:
OK I see what you are referring to:

> To create a new tag, type a version number for your release, then click Create new tag.

Well, lets test this first and if that works, les document this in more detail.

Reply:
I used it before, it works. But I can add a note as well.

Reply:
Right, I believe them it works, but I want to verify whether and how it's signing the tag.

Reply:
Are we signing tags? It looks similar to this, if the commit is signed, it shows a green tick https://github.com/modcluster/mod_cluster/releases/tag/2.0.4.Final

Reply:
> Are we signing tags? It looks similar to this, if the commit is signed, it shows a green tick https://github.com/modcluster/mod_cluster/releases/tag/2.0.4.Final

Doing a quick check - actually we aren't signing tags - because our maven plugin configuration doesn't sign them on tagging although the feature is available since 2018 (https://issues.apache.org/jira/browse/SCM-486)